examples/rhoai-quickstart/feast-demo-quickstart.ipynb

```diff
@@ -0,0 +1,1530 @@
+    "\n",
+    "print(\"----- Feature schema -----\\n\")\n",
+    "print(training_df.info())\n",
+    "\n",
+    "print()\n",
+    "print(\"----- Example features -----\\n\")\n",
+    "print(training_df.head())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "70322893-947f-46cf-a4a8-6a49d8c799b5",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "## Run offline inference (batch scoring)\n",
+    "To power a batch model, we primarily need to generate features with the get_historical_features call, but using the current timestamp"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "id": "c1fccc14-9d94-4d1a-a91a-d8b82f8d8044",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "WARNING:root:_list_feature_views will make breaking changes. Please use _list_batch_feature_views instead. _list_feature_views will behave like _list_all_feature_views in the future.\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n",
+      "----- Example features -----\n",
+      "\n",
+      "   driver_id                  event_timestamp  \\\n",
+      "0       1001 2024-09-24 03:08:48.536177+00:00   \n",
+      "1       1002 2024-09-24 03:08:48.536177+00:00   \n",
+      "2       1003 2024-09-24 03:08:48.536177+00:00   \n",
+      "\n",
+      "   label_driver_reported_satisfaction  val_to_add  val_to_add_2  conv_rate  \\\n",
+      "0                                   1           1            10   0.102868   \n",
+      "1                                   5           2            20   0.691400   \n",
+      "2                                   3           3            30   0.914943   \n",
+      "\n",
+      "   acc_rate  avg_daily_trips  conv_rate_plus_val1  conv_rate_plus_val2  \n",
+      "0  0.695487              361             1.102868            10.102868  \n",
+      "1  0.798216               18             2.691400            20.691400  \n",
+      "2  0.274195              368             3.914943            30.914943  \n"
+     ]
+    }
+   ],
+   "source": [
+    "entity_df[\"event_timestamp\"] = pd.to_datetime(\"now\", utc=True)\n",
+    "training_df = store.get_historical_features(\n",
+    "    entity_df=entity_df,\n",
+    "    features=[\n",
+    "        \"driver_hourly_stats:conv_rate\",\n",
+    "        \"driver_hourly_stats:acc_rate\",\n",
+    "        \"driver_hourly_stats:avg_daily_trips\",\n",
+    "        \"transformed_conv_rate:conv_rate_plus_val1\",\n",
+    "        \"transformed_conv_rate:conv_rate_plus_val2\",\n",
+    "    ],\n",
+    ").to_df()\n",
+    "\n",
+    "print(\"\\n----- Example features -----\\n\")\n",
+    "print(training_df.head())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "56bd764f-4455-4ade-b630-81fdbb2eb444",
+   "metadata": {},
+   "source": [
+    "## Ingest batch features into your online store"
```

Comment:
No further explanation?

Reply:
I have added more details now. Please let me know if it is not very clear.